RELEASING.md

```diff
@@ -2,10 +2,12 @@
 
 1. `git checkout master`
 1. `git pull origin master`
+1. `git checkout -b version-bump`
 1. `npm version x.x.x`
-1. `git push`
-1. `git push --tags`
+1. `git push origin version-bump`
 1. Ensure tests have passed on that tag
+1. Merge PR
+1. `git push --tags`
```

Comment:
With this moved down, it probably makes sense to remove the "Ensure tests have passed on that tag" instruction?  Or update the "Merge PR" add check tests pass and merge?

Reply:
Yep, good call. Updated.